code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -120,6 +120,26 @@
 				/obj/item/weapon/wrench = 4,
 				/obj/item/weapon/relic = 3,
 				/obj/item/weaponcrafting/reciever = 1,
+				/obj/item/clothing/head/cone = 2,
+				/obj/item/weapon/skub = 1,
+				/obj/item/weapon/grenade/smokebomb = 2,
+				/obj/item/device/geiger_counter = 3,
+				/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/orange = 1, //only for the best
+				/obj/item/device/radio/headset = 1,
+				/obj/item/device/assembly/infra = 1,
+				/obj/item/device/assembly/igniter = 2,
+				/obj/item/device/assembly/signaler = 2,
+				/obj/item/device/assembly/mousetrap = 2,
+				/obj/item/weapon/reagent_containers/syringe = 2,
+				/obj/item/clothing/gloves/color/orange = 1,

```

Comment:
ok so nobody can give me a straight answer how to do that I guess I'm not for now